Composer/packages/client/src/pages/botProject/skill-configuration/IsSkill.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import React from 'react';
+import { useRecoilValue } from 'recoil';
+import { Toggle } from 'office-ui-fabric-react/lib/components/Toggle';
+import cloneDeep from 'lodash/cloneDeep';
+import formatMessage from 'format-message';
+
+import { dispatcherState, rootBotProjectIdSelector, settingsState } from '../../../recoilModel';
+import { mergePropertiesManagedByRootBot } from '../../../recoilModel/dispatchers/utils/project';
+
+export const toggle = css`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+`;
+
+type Props = {
+  projectId: string;
+};
+
+export const IsSkill: React.FC<Props> = ({ projectId }) => {
+  const { setSettings } = useRecoilValue(dispatcherState);
+  const rootBotProjectId = useRecoilValue(rootBotProjectIdSelector) || '';
+  const settings = useRecoilValue(settingsState(projectId));
+  const mergedSettings = mergePropertiesManagedByRootBot(projectId, rootBotProjectId, settings);
+  const { skillConfiguration } = mergedSettings;
+
+  const setIsSkill = React.useCallback(
+    (isSkill: boolean) => {
+      const updatedSetting = {
+        ...cloneDeep(mergedSettings),
+        skillConfiguration: { ...skillConfiguration, isSkill },
+      };
+      setSettings(projectId, updatedSetting);
+    },
+    [mergedSettings, projectId, skillConfiguration]
+  );
+
+  const toggleIsSKill = React.useCallback(
+    (event, checked?: boolean) => {
+      setIsSkill(!!checked);
+    },
+    [setIsSkill]
+  );
+
+  return (
+    <React.Fragment>
```

Comment:
Jsx requires React.Fragment
![image](https://user-images.githubusercontent.com/17039790/112337828-e4abc980-8c83-11eb-8b91-fac43f32bf1a.png)

Reply:
<> and <Reat.Fragment> are the same, <> is shorthand for it. Here I'm saying you dont need it at all. The lint is telling you you can't use shorthand <> because of emotion jsx limit

Reply:
Oh, good point. The contents are wrapped in a div.